src/types.json

```diff
@@ -75,28 +75,39 @@
   "EpochNo": "u32",
   "EpochLen": "u32",
   "SlotNo": "u64",
+  "Balance": "u64",
   "EpochDetail": {
     "validator_count": "u8",
     "starting_slot": "SlotNo",
     "expected_ending_slot": "SlotNo",
     "ending_slot": "Option<SlotNo>",
-    "total_emission": "Option<u128>",
-    "emission_for_treasury": "Option<u128>",
-    "emission_for_validators": "Option<u128>"
+    "total_emission": "Option<Balance>",
+    "emission_for_treasury": "Option<Balance>",
+    "emission_for_validators": "Option<Balance>"
   },
   "ValidatorStatsPerEpoch": {
     "block_count": "EpochLen",
-    "locked_reward": "Option<u128>",
-    "unlocked_reward": "Option<u128>"
+    "locked_reward": "Option<Balance>",
+    "unlocked_reward": "Option<Balance>"
   },
+  "Payload": {
+    "proposal": "Vec<u8>",
+    "round_no": "u64"
+  },
+  "Membership": {
+    "members": "BTreeSet<Did>",
+    "vote_requirement": "u64"
```

Comment:
Nitpick: Does it seem practical that `vote_requirement` will ever be over u8 (>255).

Reply:
u8 or u16 would have been fine too. It's set to u64 in the runtime.